googletest/test/googletest-param-test2-test.cc

```diff
@@ -37,6 +37,8 @@
 using ::testing::Values;
 using ::testing::internal::ParamGenerator;
 
+extern ParamGenerator<int> extern_gen;
```

Comment:
anon namespace, same as others

Reply:
Can't do this. This causes a build failure.